mercadopago.php

```diff
@@ -76,8 +83,34 @@ public function __construct()
         $this->confirmUninstall = $this->l('Are you sure you want to uninstall the module?');
         $this->module_key = '4380f33bbe84e7899aacb0b7a601376f';
         $this->ps_versions_compliancy = array('min' => '1.6', 'max' => _PS_VERSION_);
+        $this->path = $this->_path;
```

Comment:
não é possível utilizar o $this->_path direto?

Reply:
Não tem acesso fora do this, provavelmente é um protected

Reply:
Belezinha